db/schema.rb

```diff
@@ -59,7 +59,7 @@
   create_table "racers", force: :cascade do |t|
     t.string "first_name", null: false
     t.string "last_name", null: false
-    t.integer "gender", default: 0
+    t.integer "gender"
```

Comment:
If we don't make this change, then "male" will automatically be selected on the gender dropdown menu in the form.

Reply:
Agreed